Description: ensure that various components that use it are built against (a potentially custom) libva
Author: Olivier Tilloy <olivier.tilloy@canonical.com>

--- a/media/gpu/vaapi/BUILD.gn
+++ b/media/gpu/vaapi/BUILD.gn
@@ -23,6 +23,7 @@ generate_stubs("libva_stubs") {
 
   output_name = "va_stubs"
   deps = [ "//base" ]
+  configs = [ "//build/config/linux/libva" ]
 }
 
 source_set("vaapi") {
--- a/media/gpu/chromeos/BUILD.gn
+++ b/media/gpu/chromeos/BUILD.gn
@@ -32,6 +32,7 @@ source_set("chromeos") {
 
   if (use_vaapi) {
     deps += [ "//media/gpu/vaapi" ]
+    configs += [ "//build/config/linux/libva" ]
   }
 
   if (use_v4l2_codec) {
@@ -107,6 +108,10 @@ source_set("fourcc") {
     "//media/gpu:buildflags",
     "//media/gpu:common",
   ]
+
+  if (use_vaapi) {
+    configs += [ "//build/config/linux/libva" ]
+  }
 }
 
 source_set("video_frame_mapper") {
--- a/media/gpu/BUILD.gn
+++ b/media/gpu/BUILD.gn
@@ -160,6 +160,7 @@ component("gpu") {
       "//media/gpu/vaapi",
       "//media/gpu/vaapi:common",
     ]
+    configs += [ "//build/config/linux/libva" ]
   }
 
   if (is_win) {
--- a/content/gpu/BUILD.gn
+++ b/content/gpu/BUILD.gn
@@ -49,6 +49,10 @@ target(link_target_type, "gpu_sources")
 
   configs += [ "//content:content_implementation" ]
 
+  if (use_vaapi) {
+    configs += [ "//build/config/linux/libva" ]
+  }
+
   deps = [
     "//base",
     "//base/third_party/dynamic_annotations",
